ophyd/controls/positioner.py

```diff
@@ -425,8 +421,6 @@ def __init__(self, setpoint, readback=None,
                                         name=self.name))
 
             self._readback.subscribe(self._pos_changed)
-
-            self._set_position(self._readback.value)
```

Comment:
I'll let @dchabot clarify what he had in mind.
On Sun, Sep 20, 2015 at 5:04 PM K Lauer <notifications@github.com> wrote:

> In ophyd/controls/positioner.py
> <https://github.com/NSLS-II/ophyd/pull/144#discussion_r39934581>:
>
> > @@ -425,8 +421,6 @@ def __init__(self, setpoint, readback=None,
> >                                          name=self.name))
> >
> >              self._readback.subscribe(self._pos_changed)
> > -
> > -            self._set_position(self._readback.value)
>
> @danielballan <https://github.com/danielballan> Does ensure_connection
> check to see if it's disconnected and raise an exception, or try to connect?
>
> @dchabot <https://github.com/dchabot> I can see where that would seem a
> bit odd, but in reality the base class shouldn't know anything about the
> signals implemented in layers above it. So I don't think it's so strange to
> have to customize it for all subclasses.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NSLS-II/ophyd/pull/144/files#r39934581>.
>